Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN apt-get update && apt-get install -y \
 	aufs-tools \
 	automake \
 	bash-completion \
+	bsdmainutils \
```

Comment:
Although it isn't listed, CRIU won't work without this. I believe the thing required might be "hexdump" but I'd have to dig more to figure it out exactly.